src/System.Device.Gpio.Tests/GpioControllerTestBase.cs

```diff
@@ -31,6 +32,7 @@ public void ControllerCanTurnOnLEDs()
         [Fact]
         public void PinValueReturnsToLowAfterDispose()
         {
+            Console.WriteLine("PinValueReturnsToLowAfterDispose run");
```

Comment:
can we make the tests only output content on failure?  a passing test shouldn't fill the screen with output.

Reply:
These Console writing rows committed accidentally, will be removed